src/controller/ensMetadata.ts

```diff
@@ -97,6 +100,8 @@ export async function ensMetadata(req: Request, res: Response) {
         created_date: 1580346653000,
         tokenId: '',
         version: Version.v1,
+        // add timestamp of the request date
+        last_request_date: Date.now()
```

Comment:
maybe add `let last_request_date =  Date.now()` outside of try so that it only calls  `Date.now()` in one place?